core/src/banking_stage/consumer.rs

```diff
@@ -507,6 +507,14 @@ impl Consumer {
         self.qos_service.accumulate_actual_execute_cu(cu);
         self.qos_service.accumulate_actual_execute_time(us);
 
+        let (committed_cu, committed_us, adjust_cu) =
+            Self::accumulate_commit_transactions_result(commit_transactions_result.as_ref().ok());
+        self.qos_service
+            .accumulate_committed_execute_cu(committed_cu);
+        self.qos_service
+            .accumulate_committed_execute_time(committed_us);
+        self.qos_service.accumulate_committed_adjust_cu(adjust_cu);
```

Comment:
i think the following style is more readable and easy to optimization out.

nits: 

```
if let Ok(...) = commit_transactions_result.as_ref() {
    let (committed_cu, ...) = Self::accumulate_commit_transactions_result(...);
}